code_of_conduct.md

```diff
@@ -0,0 +1,37 @@
+# Code of Conduct
+
+The Women Who Code DC Mission has three main goals:
+1. That women software engineers feel confident and empowered to pursue their goals.
+1. That every woman codes
```

Comment:
I agree with sir jess the brave. 
I might also add that not every woman who codes is a software engineer, so maybe make line 4 more general to coding and not a specific application of coding?